linode/nbconfigs/tmpl/data_basic.gotf

```diff
@@ -3,6 +3,7 @@
 {{ template "nb_configs_data_base" . }}
 
 data "linode_nodebalancer_configs" "foo" {
+    depends_on = [ linode_nodebalancer_config.foo, linode_nodebalancer_config.bar ]
```

Comment:
This will be a good addition to other tests that depends on a particular resource 👍

Reply:
Thanks for the suggestion! I just added it to the data source filtering test case. Does it look good?